Description: Delete sandbox files after use.
 .
 cms (0.9~git20130120+1~eec7dca4-1) precise; urgency=low
 .
   * Initial release.
Author: Bernard Blackham <b-pkg@largestprime.net>

---

--- cms-0.9~git20130120+1~eec7dca4.orig/examples/cms.conf.sample
+++ cms-0.9~git20130120+1~eec7dca4/examples/cms.conf.sample
@@ -49,7 +49,7 @@
     "_help": "Don't delete the sandbox directory under /tmp/ when they",
     "_help": "are not needed anymore. Warning: this can easily eat GB",
     "_help": "of space very soon.",
-    "keep_sandbox": true,
+    "keep_sandbox": false,
 
 
 
